src/qml/qmlplayermanagerproxy.cpp

```diff
@@ -31,6 +32,20 @@ QmlPlayerProxy* QmlPlayerManagerProxy::getPlayer(const QString& group) {
             [this, group](const QString& trackLocation, bool play) {
                 loadLocationToPlayer(trackLocation, group, play);
             });
+    connect(pPlayerProxy,
+            &QmlPlayerProxy::loadTrackRequested,
+            this,
+            [this, group](TrackPointer track,
+#ifdef __STEM__
```

Comment:
why are we ifdeffing stems? Can we assume that by the time QML exists, every build will have stems?  if so, then I'm ok making stems a prerequisite for building qml.  (but not if this would force *everyone* to have stems turned on)

Reply:
QML is already expected to be enabled for controller screens rendering, since 2.6. We could expect that both `QML` and `STEM` might be stable enough when we reach 3.0 to remove these two flags tho, but for now they are needed.

Reply:
@ywwg Do you agree with keeping these ifdefs for now?